samples/download_view_image.py

```diff
@@ -43,7 +43,7 @@ def main():
     tableau_auth = TSC.TableauAuth(args.username, password, site_id=site_id)
     server = TSC.Server(args.server)
     # The new endpoint was introduced in Version 2.5
-    server.version = 2.5
+    server.version = "2.5"
```

Comment:
I ran this as a test; turns out there was a small bug in that this needs to be a string.

We can file a new issue on making that a `@property.setter` and more robust to that kind of error